packages/@azure/keyvault/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
-    "module": "es6",
+    "module": "commonjs",
```

Comment:
Will not want this change. It's for get the sample running under ts-node.  `cross-env` would be a better option to pass this to ts-node.